lib/timer.py

```diff
@@ -117,5 +112,17 @@ def run_jobs(cfg: UAConfig, current_time: datetime):
 if __name__ == "__main__":
     cfg = UAConfig()
     current_time = datetime.utcnow()
-    setup_logging(logging.INFO, logging.DEBUG)
+
+    setup_logging(
+        logging.CRITICAL,
+        logging.DEBUG,
+        log_file="/var/log/ubuntu-advantage-timer.log",
+        logger=LOG,
+    )
+    LOG.propagate = False
```

Comment:
Do we still need this call ?

Reply:
I think we can change the whole configuration to the root logger now that we are going for a separate file. Will update.